DenisZackharov/teacherss
# Add company model

### Summary

None.

### How it works

None.

### Test plan

- execute app run `bin/dev`
- create seeds run `bin/rails db:seed:developement`
- open localhost and sign in with credentials:
  'director@example.com'
  '123456'
- you will see company name


# Checklist:


### Deploy notes

Notes regarding deployment the contained body of work.
These should note any db migrations, ENV variables, services, scripts, etc.
